### DISCUSSION TITLE
Ability to Modify Schedules In The Dagster UI

### DISCUSSION THREAD

**Discussion by mathenymi** [dagster-io/dagster#21290] [Ideas]
Hello! 

If this hasn't been requested/suggested already. I think it would be phenomenal to be able to modify the schedule of a job from the UI. Would help a lot when we have to modify a schedule on a deployed system. 

Thanks!

**Comment by JorisGanne** [comment]
We have implemented this with environmental variables which can easily be changed within the UI. So, you can use an environmental variable as part of the schedule definition. In our deployment, this works completely fine.

**Reply by mathenymi** [reply]
@JorisGanne Do you have a link for documentation or an example of this?

**Reply by JorisGanne** [reply]
@MathenyCode here is an example assuming that your have an environmental variable CRON_SCHEDULE
ScheduleDefinition(
    job=full_load, cron_schedule= os.getenv("CRON_SCHEDULE"), execution_timezone="Europe/Brussels")
